tests/test_write_pcapng.c

```diff
@@ -35,33 +35,112 @@ int main(int argc, const char** args) {
 	const char* outfile = args[1];
 	light_pcapng writer = light_pcapng_open(outfile, "wb");
```

Comment:
You will need to commit the new `tests/results/test_write_pcapng.pcapng` file that got generated from this test

Reply:
the file you committed doesn't match the one created by CI.
either you did not run the tests before committing the pcapng file
or the changes you added are writing random data to the pcapng file (missing memory initialization)